src/main/scala/cromwell/engine/ExecutionStore.scala

```diff
@@ -27,7 +27,12 @@ class ExecutionStore(binding: WdlBinding) {
     for {
       callEntry <- table if callEntry._2 == ExecutionStatus.NotStarted
       call = callEntry._1
-      if call.prerequisiteCalls().forall(table.get(_).get == ExecutionStatus.Done)
+      if call.prerequisiteCalls().forall{
+        case c:Call => table.get(c).get == ExecutionStatus.Done
+        case x =>
+          println(s"runnableCalls: $x")
+          false
```

Comment:
@scottfrazer Sorry for singling out an unfinished block of code to be the cloud I'm shaking my fist at. This is something I could have chosen almost any PR from any of us to finally start yammering on about the topic.